app/Http/Controllers/SiteController.php

```diff
@@ -448,7 +448,11 @@ public function viewSite(int $siteid): View
 
         $xml = begin_XML_for_XSLT();
 
-        $projectid = (int) $_GET['project'];
+        $projectid = 0;
+        if(isset($_GET['project'])) {
+            $projectid = (int) $_GET['project'];
+        }
```

Comment:
This is a great opportunity to use the null coalescing operator for better readability!

```suggestion
        $projectid =  (int) ($_GET['project'] ?? 0);
```